test/extended/openstack/servergroup.go

```diff
@@ -79,7 +80,23 @@ var _ = g.Describe("[sig-installer][Feature:openstack] The OpenStack platform",
 
 			serverGroup, err := servergroups.Get(computeClient, serverGroupsWithThatName[0]).Extract()
 			o.Expect(serverGroup.Members, err).To(o.ContainElements(masterInstanceUUIDs...))
+
+			if len(serverGroup.Policies) == 1 {
+				//host_ids := make([]string, 0)
+				host_ids := make(map[string]int)
+
+				for _, server_id := range masterInstanceUUIDs {
+					server, err := servers.Get(computeClient, server_id.(string)).Extract()
+					o.Expect(err).NotTo(o.HaveOccurred())
+					host_ids[server.HostID] += 1
+				}
+				if serverGroup.Policies[0] == "strict-anti-affinity" {
```

Comment:
as far as I'm aware the "strict" anti-affinity policy is really called `anti-affinity` in OpenStack

Reply:
Question: what would you think about checking the `install-config`'s setting instead of the OpenStack server group setting? I know it's supposed to be the same, but checking the install-config seems more endtoendish to me. WDYT?

Reply:
> as far as I'm aware the "strict" anti-affinity policy is really called `anti-affinity` in OpenStack

yes - you are right :-)

Reply:
> Question: what would you think about checking the `install-config`'s setting instead of the OpenStack server group setting? I know it's supposed to be the same, but checking the install-config seems more endtoendish to me. WDYT?

I think it's better